src/main/java/uk/gov/hmcts/dts/fact/model/Contact.java

```diff
@@ -30,26 +31,34 @@ public Contact(uk.gov.hmcts.dts.fact.entity.Contact contact) {
 
     private String getDescription(final uk.gov.hmcts.dts.fact.entity.Contact contact) {
         final ContactType contactType = contact.getContactType();
-        String contactDescription = (contactType == null) ? contact.getName() : contactType.getName();
+        final String contactDescription = (contactType == null) ? contact.getName() : contactType.getName();
 
-        if (contact.isFax() && !contactDescription.equalsIgnoreCase(FAX)) {
-            return new StringBuilder(contactDescription)
-                .append(" ")
-                .append(FAX.toLowerCase(Locale.getDefault()))
-                .toString();
+        if (contact.isFax()) {
```

Comment:
This looks radiant! Can I check if unit tests have been written for this bit? I can't find them anywhere for the Contact model unit tests